core-sdk-samples/higgs-shop-sample-app/README.md

```diff
@@ -21,25 +21,116 @@ By using this sample app, you will quickly learn how to:
 
 2. Install package dependencies using `npm install`
 
-3. Run the project using `npm start`
+3. In the root of this project, rename `.env.sample` to `.env` .
+
+4. Visit your [mParticle Workspace](https://app.mparticle.com/setup/inputs/apps), then navigate to **Setup > Inputs** in the left column to generate a Web API key.
```

Comment:
I would make 4 a subitem of 5 or rephrase the step to "If you don't have an mParticle API Key, do this...."